### ISSUE TITLE
Run the fuzz test vector with `--runs=1` in CI

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/rs-soroban-sdk#1363]
Run the fuzz test vector with `--runs=1` in CI to make sure it continues to work.

The code doesn't get executed in CI so it can quickly become out of date.

**Comment by graydon** [comment]
@leighmcculloch yes and what I'm saying is that https://github.com/stellar/stellar-core/pull/5121 includes a change to the _CI unit testsuite_ of core such that it will always do a smoke-test run of every fuzz harness in both core and soroban. Not under a fuzzer specifically, just under a PRNG that cranks for a few hundred iterations to make sure the target still works.

So like .. I guess technically there's a small window between breaking a fuzz harness here and someone noticing the next time it gets picked up with a core soroban bump, but that's it. Otherwise it'll be noticed. If you want to close that gap too I guess we can leave this open. But my point is that it's not likely any soroban fuzz harness will be broken for long once https://github.com/stellar/stellar-core/pull/5121 lands.

**Comment by leighmcculloch** [comment]
👍🏻 Yup sounds good. This issue isn't about a fuzz harness or a real fuzz target, it's about a test in this repo that checks that developers can use the soroban-sdk to do fuzzing.
